ompi/runtime/ompi_mpi_params.c

```diff
@@ -115,6 +116,14 @@ int ompi_mpi_register_params(void)
                                  MCA_BASE_VAR_SCOPE_READONLY,
                                  &ompi_mpi_yield_when_idle);
 
+    ompi_mpi_sleep_when_idle_threshold = -1;
+    (void) mca_base_var_register("ompi", "mpi", NULL, "sleep_when_idle_threshold",
+                                 "Sleep after waiting for MPI communication too long",
```

Comment:
It's weird to put "MPI" in the help message for an OPAL MCA var...  Can this be re-worded?

Reply:
this is basically a copy/paste of ```yield_when_idle```, i fixed both